tests/kernel/condvar/condvar_api/src/main.c

```diff
@@ -557,6 +557,10 @@ void _condvar_usecase(long multi)
 
 }
 
+/* Skip these two board when trying to use CONFIG_MP_NUM_CPUS > 1
+ * please see issue #33558
+ */
+#if !defined(CONFIG_BOARD_QEMU_CORTEX_A53) && !defined(CONFIG_X86_64)
```

Comment:
It's cleaner to write this inline instead of #if'ing the whole test function, e.g.:

```c
 void test_condvar_usecase_signal(void)
{
    if (IS_ENABLED(CONFIG_BOARD_QEMU_CORTEX_A53)
        || IS_ENABLED(CONFIG_X86_64)) {
        /* See #33558  */
        ztest_test_skip();
    }
    _condvar_usecase(0);
}
```

Reply:
Got it, I have refined it based on your suggestion, thanks for pointing out this!

Reply:
got learned again, thanks!